dev/archery/archery/cli.py

```diff
@@ -318,9 +318,9 @@ def lint(ctx, src, fix, iwyu_all, **checks):
 @click.option("--src", metavar="<arrow_src>", default=None,
               callback=validate_arrow_sources,
               help="Specify Arrow source directory")
-@click.option("--whitelist", "-w", help="Allow only these rules")
-@click.option("--blacklist", "-b", help="Disallow these rules")
-def numpydoc(src, symbols, whitelist, blacklist):
+@click.option("--enable-rule", "-e", multiple=True, help="Allow only these rules")
```

Comment:
Are there docs that need to be updated for this change? Crossbow/GHA jobs that need to be called differently?

Reply:
No, there are not.